packages/f/filament/xmake.lua

```diff
@@ -1,42 +1,70 @@
 package("filament")
-
     set_homepage("https://google.github.io/filament/")
     set_description("Filament is a real-time physically-based renderer written in C++.")
     set_license("Apache-2.0")
 
     if is_plat("windows") and is_arch("x64") then
-        add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-windows.tgz")
+        add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-windows.tgz",
+                 "https://github.com/google/filament/releases/download/v$(version)/filament-windows.tgz")
         add_versions("1.9.23", "8f5728dd944c052f991bfb15d18cd6fc8f678d0cbd37bb066d76bf682ab789c8")
         add_versions("1.20.3", "0a3fdd5fe8662a02117f3de51dcbea3b260cff716a7cffa407ca939727d7b634")
         add_versions("1.32.0", "6372895cd7729df722bb0529a1ceb086aee3e532268d541db88aba4a252cde4e")
+        add_versions("1.67.0", "07acead9d2fc2c19058656da679fe33078f4de82d7a762cdab4af6c9e02d05b2")
+        add_versions("1.67.1", "f7d6dcd5d848835ae83ab908cdff001cb5c3c65f33bc89d036c77fba143572c7")
     elseif is_plat("macosx") and is_arch("x86_64") then
         add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-mac.tgz")
         add_versions("1.9.23", "0adbf2359338e28a80b2ef84c70d8914b56ed1c97ef0135603fcd330ec4c34a1")
         add_versions("1.20.3", "820f2c7b5360021b9ff361f0868b45613726d6704f9112e8c8cf92d07c7c95b7")
         add_versions("1.32.0", "f8bd877227f10a9dc10513bfc6a396a5bb0c65b15d2fd771e2e5de4d39267395")
+        add_versions("1.67.0", "1f0db35a7808d944fdd65b321b99093969866869f7abf2ed5386615127880db0")
+        add_versions("1.67.1", "edd708241f146f216c3a8ef93e0eb5cc16e13f0ae95c35c83daefde4f3546f2b")
+    elseif is_plat("macosx") and is_arch("arm64") then
+        add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-mac.tgz")
+        add_versions("1.67.0", "1f0db35a7808d944fdd65b321b99093969866869f7abf2ed5386615127880db0")
+        add_versions("1.67.1", "edd708241f146f216c3a8ef93e0eb5cc16e13f0ae95c35c83daefde4f3546f2b")
     elseif is_plat("linux") and is_arch("x86_64") then
         add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-linux.tgz")
```

Comment:
Precompiled binaries for Linux have poor compatibility across different distributions. This package is compiled using libc++ (maybe clang is used), Perhaps they changed the compiler in the new version.

but on Linux CI, gcc uses libstdc++ by default, which can cause linking issues. This can also occur on the user side.

>/home/runner/.xmake/packages/f/filament/1.67.0/5c37058f133a4567b518692be9ac878a/lib/x86_64/libfilament.a(Fence.cpp.o): in function `filament::FFence::FFence(filament::FEngine&)::$_0::~$_0()':
Fence.cpp:(.text._ZZN8filament6FFenceC1ERNS_7FEngineEEN3$_0D2Ev+0x2b): undefined reference to `std::__1::__shared_weak_count::__release_weak()'